modules/hello-world/pages/sample-application.adoc

```diff
@@ -1,9 +1,180 @@
-= Sample App Tutorial
+= Sample Application
+:nav-title: Travel Sample App
 :content-type: tutorial
+:page-topic-type: tutorial
 :page-aliases: ROOT:sample-application,ROOT:tutorial4,ROOT:sample-app-backend
 
+[abstract]
+Discover how to program interactions with the Couchbase Server via the data, Query, and search services -- using the Travel Sample Application with the built-in Travel Sample data Bucket.
+This version also demonstrated the Developer Preview of the upcoming Collections feature.
 
-include::ref:partial$beta-warning.adoc[]
+6.5@sdk:pages:partial$sample-application.adoc
+
+This version of the Travel Sample App uses the xref:6.5@server:developer-preview:preview-mode.adoc[Collections Developer Preview (DP)] 
```

Comment:
if this is a decision that has been agreed on widely I am okay with it too

Reply:
To be revisited post-GA